setup.cfg

```diff
@@ -12,6 +12,7 @@ disallow_incomplete_defs = True
 disallow_untyped_calls = True
 disallow_untyped_decorators = True
 disallow_untyped_defs = True
+ignore_missing_imports = True
```

Comment:
That's not good idea, to ignore every possible missing imports in the future. Change that to only ignore that one library 
`[mypy-rest_framework_simplejwt.*]
ignore_missing_imports = True`